source/postprocess/visualization.cc

```diff
@@ -678,6 +718,13 @@ namespace aspect
                              "and a factor of 8 in 3d, when using quadratic elements for the velocity, "
                              "and correspondingly more for even higher order elements.");
 
+          prm.declare_entry ("Output mesh velocity", "false",
+                             Patterns::Bool(),
+                             "For free surface computations Aspect uses an Arbitrary-Lagrangian-"
+                             "Eulerian formulation to handle deforming the domain, so the mesh "
+                             "has its own velocity field.  This may be written as an output field "
+                             "by setting output_mesh_velocity to true.");
```

Comment:
The parameter isn't actually spelled with the underscores in the last line. Since the documentation is attached to the parameter itself, maybe say "...by setting this parameter to 'true'."?